specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2020-09-01/storagesync.json

```diff
@@ -3893,6 +3920,7 @@
         },
         "pauseWaitForSyncDrainTimePeriodInSeconds": {
           "type": "integer",
+          "format": "int64",
```

Comment:
@ankushbindlish2 I think we need to change this type to int32. In previous versions of the protocol and C# SDK, it looks like it was being treated as int32 (though it wasn't explicit)